Content.Shared/Movement/Systems/SpeedModifierContactsSystem.cs

```diff
@@ -81,6 +83,11 @@ private void MovementSpeedCheck(EntityUid uid, SpeedModifiedByContactComponent c
         if (!EntityManager.TryGetComponent<PhysicsComponent>(uid, out var physicsComponent))
             return;
 
+        // entities that are weightless, i.e. no gravity, should not be affected by contact slowdowns
+        // entities that are in the air, i.e. flying mobs, should not be affected by contact slowdowns
+        if (_gravity.IsWeightless(uid) || physicsComponent.BodyStatus == BodyStatus.InAir)
```

Comment:
Ok we can have full-tile meat kudzu, rhat should slowdown in weightless state too

Reply:
I've gone ahead and fixed this interaction. In order to facilitate future code extensibility, I've added a new component that can be applied to entities that should still apply their contact slowdowns to airborne mobs. Let me know if you think this is fine or should be done in another way.

Reply:
Why component? Just bool datafield

Reply:
I overcomplicated it for myself after seeing the special component for ignoring kudzu - reimplemented fix with bool datafield. Unsure why the test is now failing though...